dvc/utils/__init__.py

```diff
@@ -212,3 +212,10 @@ def convert_to_unicode(data):
         return type(data)(map(convert_to_unicode, data))
     else:
         return data
+
+
+def tmp_postfix():
```

Comment:
You meant suffix? Also, why not just make it accept the file name as it did before?

Reply:
`os.tmpfile`, as well as `tempfile.TemporaryFile`, returns a file object, so I thought that `tmp_file(name)` returning a string would be a little bit unexpected, therefore, I extracted the logic to  generate a random suffix/postfix and the concatenation could be more explicit: `tmp_file = path + tmp_postfix()`.

The `tempfile` module supports a [gettempprefix](https://docs.python.org/2/library/tempfile.html#tempfile.gettempprefix) method, maybe renaming `tmp_postfix` to `gettempsuffix` could be more adequate.

I have no problem to rollback the changes, just sharing my train of thought :)
What do you think, @efiop ?

Reply:
@mroutis Agreed. It seems reasonable to move that "+" logic to the function itself, so you don't have to add up strings in the code each time you use that function. But I see your point about the naming. Maybe we could use something like `tmp_fname` to avoid confusion with fobj-ects? What do you think?